cmake/lfc.cmake

```diff
@@ -61,15 +61,16 @@ function(lf_build_generated_code MAIN_TARGET SOURCE_GEN_DIR)
   endif()
 
   # Check if the Include.cmake file exists in SOURCE_GEN_DIR
-  if (NOT EXISTS ${SOURCE_GEN_DIR}/Include.cmake)
+  if (NOT EXISTS ${SOURCE_GEN_DIR}/${FEDERATE}/Include.cmake)
     message(FATAL_ERROR "Include.cmake does not exist in src-gen directory: ${SOURCE_GEN_DIR}/Include.cmake")
   endif()
 
-  include(${SOURCE_GEN_DIR}/Include.cmake)
+  include(${SOURCE_GEN_DIR}/${FEDERATE}/Include.cmake)
+  message(${REACTOR_UC_PATH})
```

Comment:
Again, dont think we should introduce the dependency on this global variable. ALso no need to print the REACTOR_UC_PATH here